pkg/executor/main.go

```diff
@@ -62,13 +65,18 @@ func run() error {
 
 	logrus.Info("Writing backend config")
 
-	err = runner.WriteConfigFile()
+	path := TfWritePath
+
+	if runner.Execution.Spec.Content.Git.Path != "" {
+		path = fmt.Sprintf("%s%s", TfWritePath, runner.Execution.Spec.Content.Git.Path)
```

Comment:
Should we rather use `path.Join()` here, in order to avoid issues with extra or missing slashes?

Reply:
done